app-libs/stf/src/stf_sgx_litentry.rs

```diff
@@ -171,14 +184,14 @@ impl Stf {
 
 		//TODO change error type
 		code.ok_or_else(|| StfError::Dispatch(format!("code not found")))?;
-		let request = lc_stf_task_handler::Web2IdentityVerificationRequest {
+		let request = Web2IdentityVerificationRequest {
 			target,
 			identity,
 			challenge_code: code.unwrap(),
 			validation_data,
 			bn,
 		};
-		let xt_sender = lc_stf_task_handler::stf_task_sender::StfRequestSender::new();
+		let xt_sender = StfRequestSender::new();
```

Comment:
maybe we can simply rename it to `sender`

Reply:
done